src/commands/pull.js

```diff
@@ -0,0 +1,149 @@
+const fs = require('fs');
+const path = require('path');
+const tiged = require('tiged');
+const AdmZip = require('adm-zip');
+const semverGt = require('semver/functions/gt');
+const { get } = require('axios');
+const { flags: flagsHelper } = require('@oclif/command');
+const { ApifyCommand } = require('../lib/apify_command');
+const { success, error } = require('../lib/outputs');
+const { getLoggedClientOrThrow, getLocalConfigOrThrow, getLocalUserInfo } = require('../lib/utils');
+const { LOCAL_CONFIG_PATH } = require('../lib/consts');
+
+const extractGitHubZip = async (url, directoryPath) => {
+    const { data } = await get(url, { responseType: 'arraybuffer' });
+
+    const zipFile = new AdmZip(Buffer.from(data, 'binary'));
+
+    zipFile.extractEntryTo(zipFile.getEntries()[0].entryName, directoryPath, false);
+};
+
+class PullCommand extends ApifyCommand {
+    async run() {
+        const { args, flags } = this.parse(PullCommand);
+        const localConfig = await getLocalConfigOrThrow();
+        const userInfo = await getLocalUserInfo();
+        const apifyClient = await getLoggedClientOrThrow();
+        const cwd = process.cwd();
+
+        const isActorAutomaticallyDetected = !args?.actorId;
+        const usernameOrId = userInfo.username || userInfo.id;
+
+        const actorId = args?.actorId || localConfig?.id || `${usernameOrId}/${localConfig.name}`;
+
+        if (!actorId) throw new Error('Cannot find actor in this directory.');
```

Comment:
This will not work, since `actorId` will always be at least `"usernameOrId/undefined"`, so never falsy.